src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfFeature.java

```diff
@@ -0,0 +1,1320 @@
+package org.broadinstitute.hellbender.utils.codecs.GENCODE;
+
+import htsjdk.samtools.util.Locus;
+import htsjdk.tribble.Feature;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import scala.tools.nsc.Global;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * A GTF Feature represents one row of a GTF File.
+ * The specification of a GTF file is defined here:
+ * http://mblab.wustl.edu/GTF22.html
+ *
+ * Currently only supports version 26 or greater!
```

Comment:
The comment is out of date.  It actually supports data from versions 19 - 26.  I'll add in the user exception.